Tensile/KernelWriterAssembly.py

```diff
@@ -15400,6 +15403,8 @@ def functionSuffix(self, kernel):
       self.overflowedResources = 1
     elif self.sgprPool.size() > self.maxSgprs:
       self.overflowedResources = 2
+    elif self.agprPool.size() > self.maxAgprs:
```

Comment:
same here. It might make more sense to check kernel["EnableMatrixInstruction"]
like
   elif kernel["EnableMatrixInstruction"]  and self.agprPool.size() > self.maxAgprs: